examples/tutorials/api/estimators.py

```diff
@@ -0,0 +1,286 @@
+       for significance based on excess counts, currently available in `~gammapy.estimators.ExcessMapEstimator`.
+
+For further information on these details please refer to :doc:`/user-guide/estimators`.
+
+The setup
+---------
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy import units as u
+from IPython.display import display
+from gammapy.datasets import SpectrumDatasetOnOff, Datasets, MapDataset
+from gammapy.estimators import (
+    FluxPointsEstimator,
+    ExcessMapEstimator,
+    FluxPoints,
+)
+from gammapy.modeling import Fit
+from gammapy.modeling.models import SkyModel, PowerLawSpectralModel
+from gammapy.utils.scripts import make_path
+
+
+######################################################################
+# Flux Points Estimation
+# ----------------------
+#
+# We start with a simple example for flux points estimation taking multiple datasets into account.
+# In this section we show the steps to estimate the flux points.
+# First we read the pre-computed datasets from `$GAMMAPY_DATA`.
+#
+
+datasets = Datasets()
+path = make_path("$GAMMAPY_DATA/joint-crab/spectra/hess/")
+
+for filename in path.glob("pha_obs*.fits"):
+    dataset = SpectrumDatasetOnOff.read(filename)
+    datasets.append(dataset)
+
+######################################################################
+# Next we define a spectral model and set it on the datasets:
+#
+
+pwl = PowerLawSpectralModel(index=2.7, amplitude="5e-11  cm-2 s-1 TeV-1")
+datasets.models = SkyModel(spectral_model=pwl, name="crab")
+
+######################################################################
+# Before using the estimators, it is necessary to first ensure that the model is properly
+# fitted. This applies to all scenarios, including light curve estimation. To optimize the
+# model parameters to best fit the data we utilise the following:
+#
+
+fit = Fit()
+fit_result = fit.optimize(datasets=datasets)
+print(fit_result)
+
+######################################################################
+# A fully configured Flux Points Estimation
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The `~gammapy.estimators.FluxPointsEstimator` estimates flux points for a given list of datasets,
+# energies and spectral model. The most simple way to call the estimator is by defining both
+# the ``source`` and ``energy_edges``.
+# Here we prepare a full configuration of the flux point estimation.
+# Firstly we define the ``backend`` for the fit:
+#
+
+fit = Fit(
+    optimize_opts={"backend": "minuit"},
+    confidence_opts={"backend": "scipy"},
+)
+
+
+######################################################################
+# Define the fully configured flux points estimator:
+#
+
+energy_edges = np.geomspace(0.7, 100, 9) * u.TeV
+
+fp_estimator = FluxPointsEstimator(
+    source="crab",
+    energy_edges=energy_edges,
+    n_sigma=1,
+    n_sigma_ul=2,
+    selection_optional="all",
+    fit=fit,
+    reoptimize=False,
```

Comment:
Please show how to configure the norm here. That is often required for upper limit calculations